### ISSUE TITLE
popup not refreshed when click on users in meteor 0.8.2

### ISSUE THREAD

**Issue by dougserfass** [alanning/meteor-modal-example#2]
in manage-roles.js, Template.editRolesForm.rendered is only called the first time the template is rendered
as a result, in meteor 0.8.2, popup is not refreshed when click on users
in other words...popup is only correct for first user clicked

**Comment by alanning** [comment]
Right, the example is only valid for pre-blaze meteor (due to how template rendering has changed).

If you'd like to submit a PR with blaze compatibility that would be a big help!

**Comment by dougserfass** [comment]
i fixed the problem in our app for now by brute force...i do a location.reload() after cancel/save on popup...not a good solution, but our app is a prototype
what specifically do i do to submit a problem report with blaze compatibility other than to say the sample app popup does not work correctly with blaze?

**Comment by alanning** [comment]
Ah, PR generally stands for pull request.  :-)

It's a github thing where you can send updated code to fix issues or add features.

**Comment by dougserfass** [comment]
sorry for not understanding all of the github lingo
also, let me say that we could not have created our app in such a short time without meteor-roles and meteor-modal-example
i am not sure you would want to integrate my blaze compatibility fix (adding location.reload() to cancel/save) into your project as it really is not a long term solution
